File: java/grpc-demo-client/src/main/java/com/grpc/demo/lqm/ClientApp.java
package com.grpc.demo.lqm;

import com.grpc.demo.lqm.client.HelloClient;
import io.grpc.ManagedChannel;
import io.grpc.ManagedChannelBuilder;

public class ClientApp {
    private static final String TARGET = "localhost:3000";
    private static final String NAME = "liqiming java";

    public static void main(String[] args) {
        final ManagedChannel channel = ManagedChannelBuilder.forTarget(TARGET).usePlaintext().build();
        final HelloClient helloClient = new HelloClient();
        helloClient.init(channel);
        helloClient.sayHello(NAME);
    }
}

File: node/dynamic_code/client/clientApp.js
const { init, grpc } = require('proto');
const helloServiceProto = init('hello_service.proto').hello;
const target = 'localhost:3000';
const client = new helloServiceProto.HelloService(target,
    grpc.credentials.createInsecure());

client.SayHello({ name: "liqiming node dy" }, function (err, response) {
    console.log('Greeting:', response.message);
});